src/main/kotlin/org/rust/cargo/runconfig/buildtool/Utils.kt

```diff
@@ -85,7 +87,10 @@ class MockBuildProgressListener(buildsCount: Int = 1) : BuildProgressListener {
     }
 
     @Throws(InterruptedException::class)
-    fun waitFinished(timeout: Long = 1, unit: TimeUnit = TimeUnit.MINUTES) {
-        latch.await(timeout, unit)
+    fun waitFinished(timeoutMs: Long = 1000) {
```

Comment:
One second doesn't looks enough for these tests

Reply:
That's why it was changed to 5000 (for a start) in the next commit